jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantDisplay/model.js

```diff
@@ -25,16 +25,16 @@ export default jbrowse => {
 
     .actions(self => ({
       selectFeature(feature) {
-        const extendedVariantDisplayConfig = getContainingTrack(self).configuration.metadata.value.extendedVariantDisplayConfig
         const trackId = getContainingTrack(self).configuration.trackId
         const session = getSession(self)
+        const metadata = getContainingTrack(self).configuration.metadata.value
```

Comment:
I think we should still put all our widget-specific properties within one object, like metadata.extendedVariantDisplayConfig. The point is that we cant control what other code uses metadata, so we should sandbox our config to the highest degree we can. It's possible this means we could adjust our original config properties to be metadata.extendedVariantDisplayConfig.fieldGroups (or something like that).